console/frontend/src/main/frontend/src/app/views/webservices/webservices.service.ts

```diff
@@ -5,13 +5,13 @@ import { AppService } from 'src/app/app.service';
 
 export interface Service {
   name: string;
-  method: string;
+  methods: string;
```

Comment:
Dit moet volgens mij een stringarray worden? @philipsens

Reply:
I'm not sure what the backend produces (looks like a comma separates string), but I'd say, ideally, yes.

I've checked out this PR, and it looks like the functionality is broken.

For example, the backend returns this:
```js
adapter : "RestListener_GetDoc" method : "GET" name : "RestListener_GetDoc" receiver : "RestListener [RestListener_GetDoc]" uriPattern : "doc"
```

So changing the model, won't do any good, unless it is also changed on the backend.

Reply:
@philipsens, yes I've changed the backend and named it plural. That's why I am changing it now. 

Secondly, I've tested it and both values are displayed in the table: `POST,PUT`.

I will try the array notation too, and if that also works, I'll use that. 
So, `methods: string[]`, I guess.

Reply:
I wonder why my backup didn't get updated, I think I didn't update the docker image correctly (still getting used to intellij maven)

Reply:
I usually do `rm -Rf docker/**/target` manually and only run the `mvn package -DskipTests -Pdocker,Tomcat` to not clean all Java files. That should work fine and fast, because it does not through away all the cached output of previous builds. 
Thanks for the suggestion. I'll try.

Reply:
In my case, that wasn't the problem. I used the Maven client in IntelliJ but pressed `Frank!Framework - Docker > Tomcat > Test`, so it only builds test and used an older `frankframework` base image.